src/ContainersRunImageModal.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+class ContainersRunImageModal extends React.Component {
+    constructor(props) {
+      super(props);
+
+    }
```

Comment:
Unnecessary if you don't do anything other than calling the parent. Also, this component can be functional, it doesn't have any state.